modules/end-user-guide/partials/proc_setting-up-che-editor-yaml.adoc

```diff
@@ -0,0 +1,23 @@
+[id="setting-up-che-editor-yaml_{context}"]
+= Setting up che-editor.yaml
+
+To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. This is convenient when, as a developer using {prod-short}, you clone your personal fork of the project repository into your {prod-short} user's workspaces, and this way you don't have to specify your IDE every time you start another workspace with a clone of that repository.
```

Comment:
```suggestion
To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, use the `che-editor.yaml` file. This is convenient for developers using {prod-short} who clone their personal fork of the project repository into their {prod-short} user's workspaces. This way you don't have to specify your IDE every time you start another workspace with a clone of that repository.
```

Reply:
Thank you, Jana. This appears to be outdated, so I'm resolving this conversation.